website/src/pages/auth/signup.tsx

```diff
@@ -0,0 +1,92 @@
+import { Box, Button, ButtonGroup, Input, Stack } from "@chakra-ui/react";
+import Head from "next/head";
+import { FaDiscord, FaEnvelope, FaGithub, FaMagic } from "react-icons/fa";
+import { getCsrfToken, getProviders, signIn } from "next-auth/react";
+import { useRef } from "react";
+import Link from "next/link";
+
+import { AuthLayout } from "src/components/AuthLayout";
+
+export default function Signin({ csrfToken, providers }) {
+  const { discord, email } = providers;
+  const emailEl = useRef(null);
+  const signinWithEmail = () => {
+    signIn(email.id, { callbackUrl: "/", email: emailEl.current.value });
+  };
+
+  return (
+    <>
+      <Head>
+        <title>Sign Up - Open Assistant</title>
+        <meta name="Sign Up" content="Sign up to access Open Assistant" />
+      </Head>
+      <AuthLayout>
+        <Stack spacing="2">
+          {email && (
+            <Stack>
+              <Input variant="outline" size="lg" placeholder="Email Address" ref={emailEl} />
+              <Button
+                size={"lg"}
+                leftIcon={<FaEnvelope />}
+                colorScheme="gray"
+                onClick={signinWithEmail}
+                // isDisabled="false"
+              >
+                Continue with Email
+              </Button>
+            </Stack>
+          )}
+          {/* {discord && ( */}
```

Comment:
Question, why is this part commented out?

Reply:
Just to show the Discord button. It wasn't appearing with that in place, so I commented it out while I was formatting everything